Assets/Scripts/Towers/Repair logic/RepairController.cs

```diff
@@ -13,69 +14,73 @@ public enum WearState
 public class RepairController : MonoBehaviour
 {
     [SerializeField]
-    private float damageInterval;
+    private float wearInterval = 30f;
 
-    [ReadOnly, SerializeField]
-    private WearState _currentState;
+    [TextLabel(greyedOut = true), SerializeField]
```

Comment:
Is there a reason we want to avoid ReadOnly? 🤔

Reply:
No, I wouldn't say that `ReadOnly` should be avoided, just that I personally think it looks more appropriate for serialized fields that are not game objects (so mainly fields of primitive types, like `int`, `string`, or `enum` - in this case) to have the `TextLabel` attribute, as it doesn't have the input "box" around the field value. For fields that *are* game objects, it's better to use the `ReadOnly` attribute, as it enables us to click on the field value in the inspector, which will jump to the asset / game object 🤠

Try it out yourself! Which of the two attributes do you think looks better in this case? 🙂

Reply:
I see, then I definetly agree with the use of `TextLabel` 👌